app/controllers/avo/actions_controller.rb

```diff
@@ -12,9 +12,15 @@ def show
 
     def handle
       resource_ids = action_params[:fields][:resource_ids].split(",")
-      models = @resource.class.find_scope.find resource_ids
+      selected_all = action_params[:fields][:selected_all].to_i == 1
 
-      fields = action_params[:fields].except("resource_ids")
+      models = if selected_all
+        ActiveRecord::Base.connection.exec_query base64_decode_query
```

Comment:
We gotta see if this is a security issue.

Reply:
Maybe it's better to encrypt/decrypt it with a secret key?

Reply:
ActiveSupport::MessageEncryptor implemented